plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -968,11 +1078,13 @@ def main():
             scsi=dict(type='dict'),
             scsihw=dict(choices=['lsi', 'lsi53c810', 'virtio-scsi-pci', 'virtio-scsi-single', 'megasas', 'pvscsi']),
             serial=dict(type='dict'),
+            searchdomain=dict(type='str', default=None),
             shares=dict(type='int'),
             skiplock=dict(type='bool'),
             smbios=dict(type='str'),
             snapname=dict(type='str'),
-            sockets=dict(type='int'),
+            sockets=dict(type='int', default=1),
```

Comment:
@aminvakil killed all default variables so it is probably not wise to bring them back.